interceptor/main.go

```diff
@@ -22,13 +23,23 @@ const (
 	idleTimeout  = 60 * time.Second
 )
 
-var logger = logging.InitLogger(logging.LogLevelString, "")
+var logger *zap.SugaredLogger
```

Comment:
Does this logger actually work?
The examples from zap all initialize the logger like this:
```
logger, _ := zap.NewProduction()
defer logger.Sync() // flushes buffer, if any
sugar := logger.Sugar()
```